### PR TITLE
secret distribution

### PR DESCRIPTION
**Pull request by tanalam2411**

### CODE REVIEW

**Comment by stakater-user** [comment]
@tanalam2411 Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-108-43866d87`

**Comment by stakater-user** [comment]
@tanalam2411 Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-108-a331047f`

**Comment by stakater-user** [comment]
@tanalam2411 Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-108-f40f03e4`

**Comment by stakater-user** [comment]
@tanalam2411 Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-108-a03d4c1c`

**Comment by stakater-user** [comment]
@tanalam2411 Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-108-4737fa56`

File: docs/content/sre/multi-tenant-operator/usecases/secret-distribution.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Propagate Secrets from Parent to Descendant namespaces
+
```
Comment (rasheedamir) [inline docs/content/sre/multi-tenant-operator/usecases/secret-distribution.md:2]: @tanalam2411 plz remove this extra empty line

**Comment by stakater-user** [comment]
@tanalam2411 Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-108-edcc731e`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.saap-prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.80`

**Pull request merged**